test/CMakeLists.txt

```diff
@@ -14,33 +14,23 @@ target_link_libraries(tests PRIVATE project_warnings project_options catch_main)
 # to whatever you want, or use different for different binaries
 catch_discover_tests(
   tests
-  TEST_PREFIX
-  "unittests."
-  REPORTER
-  xml
-  OUTPUT_DIR
-  .
-  OUTPUT_PREFIX
-  "unittests."
-  OUTPUT_SUFFIX
-  .xml)
+  TEST_PREFIX "unittests."
+  REPORTER xml
+  OUTPUT_DIR .
+  OUTPUT_PREFIX "unittests."
+  OUTPUT_SUFFIX .xml)
```

Comment:
This is a huge improvement in terms of readability, IMHO. Thank you for finding the `.cmake-format.yaml` parameters that permit this!